Maths/LCM.php

```diff
@@ -0,0 +1,27 @@
+<?php
+// PHP program to find LCM of two numbers
+ 
+// Recursive function to
+// return gcd of a and b
+function gcd( $a, $b)
+{
+   if ($a == 0)
+        return $b;
+    return gcd($b % $a, $a);
+}
+ 
+// Function to return LCM
+// of two numbers
+function lcm( $a, $b)
+{
+    return ($a / gcd($a, $b)) * $b;
+}
+ 
+    // Driver Code
+    $a = 15;
+    $b = 20;
+    echo "LCM of ",$a, " and "
+         ,$b, " is ", lcm($a, $b);
+ 
+// This code is contributed by cheeshan
+?>
```

Comment:
Per the [PSR-12 php style guide](https://www.php-fig.org/psr/psr-12/), the closing ?> tag must be omitted from files containing only PHP

```suggestion

```